docs/flashbots-mev-share/release-notes/2023-03.mdx

```diff
@@ -0,0 +1,19 @@
+# beta launch (March 2023)
```

Comment:
Beta launch was in April

Reply:
Nit: Keep the title formats consistent, “April 2023 (beta)” if you want to mention the beta launch